documents/final-submission-document/final-submission-document.tex

```diff
@@ -131,10 +131,37 @@ \chapter{Management Summary}
 % - Vorgehen, Technologien
 % - Ergebnisse
 % - Ausblick (optional)
+\section*{Motivation}
+%TODO: Can we just use IETF in this context?
+The IETF standard draft ``Using XMPP for Security Information Exchange'' describes how devices can exchange security-relevant information within a network, so-called ``XMPP-Grid'', using the widely adopted messaging protocol ``XMPP''.
+Such security-relevant information can be used to take protective measures automatically, e.g. to block devices running outdated software.
 
-% prototype demonstrates feasability
-% further conceptual refinement needed
-% implemenetation is complex - but doable.
+An administration interface, in the standard draft referred to as XMPP-Grid Broker, is used to configure an XMPP-Grid.
+Today, no such XMPP-Grid Broker exists that is production ready and cross-platform.
+
+\section*{Project Goals, Approach}
+
+The goal of this thesis is to engineer an XMPP-Grid Broker that allows administrators to configure XMPP-Grids in a usable and productive way.
+The resulting application should only depend on the underlying standards and not on specific implementations to support the further standardisation process.
+
+A comprehensive analysis of the underlying standards is carried out in the first part of this thesis.
+Particular attention is given to portability, extensibility and security aspects in a production environment.
+By this analysis, a systematic selection of possible architecture options is executed in the form of architectural decisions.
+
+\section*{Results}
+
+The resulting implementation enables administrators to create and configure communication Topics, apprehend the hierarchy of the underlying XMPP-Grid and manage permissions of network participants.
+Additionally, persisted messages can be inspected and published on communication Topic.
+
+The XMPP-Grid Broker is implemented as Angular5 Web Application that connects directly from the web browser to the XMPP-Grid. Secure communication is assured by the use of Mutual authentication over TLS.
```

Comment:
I would have written web application lower-case, as it is an application in the web (which does not refer to a special concept of this thesis).

Would you like to keep it this way or should I change it?

Reply:
The same applies for "Mutual", but I assume this one is a typo?

Reply:
wopsy, Mutual is clearly wrong, and i agree with Web Application (which you wrote upper case in you first draft as well :wink: :laughing: ) :+1: